src/components/input/styled.js

```diff
@@ -52,7 +52,6 @@ export const StyledLabel = styled.label`
 `
 export const LabelRow = styled.div`
```

Comment:
This comment is not related to your change, but I think that when we make an update to a component that seems to be outdated we could get it up-to-date and in this case I think we could replace `div` with `Flex`.
I am adding a suggestion bellow, if you want you can merge it as is! 😄 

```suggestion
export const LabelRow = styled(Flex).attrs({
  alignItems: "center",
  color: "text",
  flex: false,
  width: "100%",
})`
  font-size: 14px;
  font-style: normal;
  font-weight: bold;
  line-height: 18px;
`
```

Reply:
Updated :)